spec/support/reports_helper.rb

```diff
@@ -3,6 +3,8 @@
 module ReportsHelper
   def run_report
     click_on "Go"
+    expect(page).to have_selector ".loading"
     perform_enqueued_jobs(only: ReportJob)
+    expect(page).to have_no_selector ".loading"
```

Comment:
I observed some flakiness at some point. But I'm not sure if this was the solution or if it was something else I did. Anyway, it usually doesn't hurt to check that the screen is updating as expected to avoid any race conditions.